### PR TITLE
[INTERNAL] Adjust documentation regarding package managers

### PR DESCRIPTION
**Pull request by flovogt**
- Remove usage of yarn
- Remove v2 specific documentation parts

### CODE REVIEW

File: docs/pages/Troubleshooting.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ Please follow our [Contribution Guidelines](https://github.com/SAP/ui5-tooling/b
 ## UI5 Project
 ### Corrupt Framework Dependencies
 
-When using the UI5 CLI in versions lower than `v2.9.1` or the `@ui5/project` module in versions lower than `v2.2.6` you might experience build- or runtime issues caused by missing or corrupt files in one or more UI5 framework dependencies. Note that this does not affect other project dependencies, such as reuse libraries, which are installed via external package managers like npm or Yarn.
+When using the UI5 CLI in versions lower than `v2.9.1` or the `@ui5/project` module in versions lower than `v2.2.6` you might experience build- or runtime issues caused by missing or corrupt files in one or more UI5 framework dependencies. Note that this does not affect other project dependencies, such as reuse libraries, which are installed via external package managers e.g. npm.
```
Comment (codeworrior) [inline docs/pages/Troubleshooting.md:15]: I wonder if this should be kept in the 3.0 documentation. it makes sense for v2, but not for v3, isn't it?

Comment (flovogt) [reply docs/pages/Troubleshooting.md:15]: Thats true. I have just went over the yarn occurrences not checking the whole section. I'll remove the whole section

Comment (flovogt) [reply docs/pages/Troubleshooting.md]: done

**Comment by d3xter666** [comment]
LGTM

**Review by matz3 — COMMENTED** [review]
LGTM

File: docs/pages/SAPUI5.md [hunk 2]
```diff
@@ -8,7 +8,7 @@
 
 ## Overview
 
-SAPUI5 libraries are hosted on the public npm registry at `registry.npmjs.org`. However, you should not install them using node package managers like npm or Yarn. Instead, please let the UI5 Tooling handle them by following this guide.
+SAPUI5 libraries are hosted on the public npm registry at `registry.npmjs.org`. However, you should not install them using node package managers e.g. npm. Instead, please let the UI5 Tooling handle them by following this guide.
```
Comment (KlattG) [inline docs/pages/SAPUI5.md:11]:
```suggestion
SAPUI5 libraries are hosted on the public npm registry at `registry.npmjs.org`. However, you should not install them using node package managers such as npm. Let UI5 Tooling handle them instead by following this guide.
```

**Review by KlattG — CHANGES_REQUESTED** [review]
Just a small suggestion, otherwise fine

**Review by matz3 — APPROVED** [review]

**Pull request merged**